test/extern/fnPtrs/passBadFnsToCChapel-cPtrToChapel.good

```diff
@@ -0,0 +1,2 @@
+passBadFnsToCChapel.chpl:11: In function 'baz':
+passBadFnsToCChapel.chpl:12: error: Can't call a C function pointer within Chapel
```

Comment:
Might be worth making that case a semantic future, so we don't lock in behavior that we aren't certain we want.